Open Judge System/Workers/OJS.Workers.ExecutionStrategies/CSharpProjectTestsExecutionStrategy.cs

```diff
@@ -76,12 +96,14 @@ public override ExecutionResult Execute(ExecutionContext executionContext)
 
             // Modify Project file
             var project = new Project(csProjFilePath);
+            var compileDirectory = project.DirectoryPath;
+            this.SetupFixturePath = $"{compileDirectory}\\{SetupFixtureFileName}.cs";
```

Comment:
Keep consistency with the project, add a runtime constant in the GlobalConstants for the file extension and reuse it.